test/odo.int.ts

```diff
@@ -12,10 +12,12 @@ import { Cluster } from '../src/openshift/cluster';
 suite('odo integration', () => {
```

Comment:
Only  changes in package.json is good to go. Everything else does not add any value .